torch/lib/TH/THDiskFile.c

```diff
@@ -556,7 +556,7 @@ static size_t THDiskFile_readString(THFile *self, const char *format, char **str
         total += TBRS_BSZ;
         p = THRealloc(p, total);
       }
-      if (fgets(p+pos, total-pos, dfself->handle) == NULL) /* eof? */
+      if (fgets(p+pos, (int) (total-pos), dfself->handle) == NULL) /* eof? */
       {
         if(pos == 0)
```

Comment:
There are warnings in the contribuild down in this file, maybe you miss some castings? Example of warning
```
/home/travis/build/pytorch/pytorch/torch/lib/TH/THDiskFile.c: In function ‘THDiskFile_new’:
/home/travis/build/pytorch/pytorch/torch/lib/TH/THDiskFile.c:618:5: warning: initialization from incompatible pointer type [-Wincompatible-pointer-types]
     THDiskFile_readChar,
     ^
```